app-dicts/myspell-sl/myspell-sl-20201231.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MYSPELL_DICT=(
+	"sl_SI.aff"
+	"sl_SI.dic"
+)
+
+MYSPELL_HYPH=(
+	"hyph_sl_SI.dic"
+)
+
+MYSPELL_THES=(
+	"th_sl_SI_v2.dat"
+	"th_sl_SI_v2.idx"
+)
+
+inherit myspell-r2
+
+DESCRIPTION="Slovenian dictionaries for myspell/hunspell"
+HOMEPAGE="https://extensions.libreoffice.org/extension-center/slovenian-dictionary-pack"
+SRC_URI="https://extensions.libreoffice.org/assets/downloads/752/pack-sl.oxt"
```

Comment:
You don't update the manifest hash for this ebuild at all. So did you install the previous version after all? I see the URL has `752` but can it be used to identify a different version? 

Rename the output file to include version info.

Reply:
Link can't identify older version because they are renamed when newer is uploaded (for eg: https://extensions.libreoffice.org/assets/downloads/z/8b7ba8bb_pack-sl.oxt) but yes, your point is good